src/main/java/org/onebusaway/alexa/lib/ObaUserClient.java

```diff
@@ -90,6 +90,17 @@ public ObaUserClient(String obaBaseUrl) throws URISyntaxException {
     }
 
     /**
+     * Returns details about a particular stop, given it's stopId
+     * @param stopId
+     * @return details about a particular stop, given it's stopId
+     */
+    public ObaStopResponse getStopDetails(String stopId) {
+        ObaStopResponse response = new ObaStopRequest.Builder(stopId).build().call();
```

Comment:
No, but that reminded me that we should be checking the HTTP code before assuming the result is valid.  Just pushed a new commit that includes this.